docs/director/03-tenancy.md

```diff
@@ -0,0 +1,37 @@
+#Tenancy
+Tenant is the object that owns resources. 
+The Compass Director is a multi-tenant service.
+
+## Tenancy in Director
+Tenancy allows the Compass Director to logically split the data. 
+Tenancy in Director is implemented on the database level.
+The tenant has its own entity stored in the database.
+Every object which belongs to a tenant has the `tenant_id` column which points to the actual tenant entity.
+A tenant is mainly described by two properties: 
+*  Global tenant identifier - can be any string, treated like identifier from an external system 
+* Internal tenant identifier - used as an internal technical identifier (UUID) to which `tenant_id` columns refer to
+Those properties are stored in the `business_tenant_mapping` table with metadata.
+Internal tenant identifier allows for unified tenant identification in Director. 
+Thanks to this approach, external systems can describe their tenants in their own way without any impact on the Director internals.
+
+Director manages the configuration of the following main objects:
```

Comment:
This is more general information than the one you start this section with. Please start the section with the most general information, such as how Director is related to tenancy, and then go into more details.